build-output/_multiple-bundles/readium-external-libs.js.bundles.js

```diff
@@ -1 +0,0 @@
-require.config({bundles: {"readium-external-libs":["IPv6","SecondLevelDomains","URIjs","console_shim","cssom","domReady","eventEmitter","jquery","jquerySizes","punycode","readium_cfi_js","underscore"]}});
```

Comment:
The pre-built Javascript bundles in the `build-output` folder should not be deleted.

Reply:
oh yep sorry not intentional!